nomos-services/data-availability/dispersal/src/backend/mod.rs

```diff
@@ -17,7 +18,7 @@ pub trait DispersalBackend {
     type NetworkAdapter: DispersalNetworkAdapter;
     type MempoolAdapter: DaMempoolAdapter;
     type Metadata: Debug + metadata::Metadata + Send;
-    type BlobId: AsRef<[u8]> + Send;
+    type BlobId: AsRef<[u8]> + Send + Copy + Serialize;
```

Comment:
This probably can be moved where we actually use it. In the implementation we do not use Copy or Serialize. Probably the other constrains could be removed as well (?)

Reply:
I have added it as the api needs to serialize return type that is dependent on the backend. The point was to return BlobId in the disperse call. I tried several approaches but it looks like that api needs to be able to serialize the response. Is there a better way to do this that I am missing?